src/components/organisms/WithNavigationBar/WithNavigationBar.scss

```diff
@@ -4,8 +4,69 @@
   padding-top: $navbar-height;
   padding-bottom: $footer-height;
 }
+
 .navbar-margin.fullscreen {
   height: 100%;
   width: 100%;
   overflow: auto;
 }
+
+.nav-logos {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-sparkle-logo,
+.nav-sparkle-logo_small {
+  position: relative;
+  margin-right: 12px;
+  &:hover {
+    width: 120px;
+  }
+  &.nav-sparkle-logo_small {
+    display: none;
+  }
+  div {
+    position: relative;
+    display: block;
+    width: 25px;
+    height: 35px;
+    background-image: url(/icons/sparkle-nav-logo.png);
+    background-size: 25px 35px;
+    &:before {
+      position: absolute;
+      content: "";
+      opacity: 0;
+      pointer-events: none;
+      left: 22px;
+      width: 108px;
+      height: 35px;
+      background-image: url(/icons/sparkle-nav-logo-text.png);
+      background-size: 108px 35px;
+      transform: translateX(-40px) scale(0.75);
+      transition: opacity 200ms, transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
```

Comment:
It would be good to make use of variables for these things instead of 'magic numbers', etc.